### DISCUSSION TITLE
对流程实例进行停止或暂停操作

### DISCUSSION THREAD

**Discussion by 405594085** [apache/dolphinscheduler#9782] [Q&A] [answered]
对流程实例进行停止或暂停操作，流程实例会一直处于准备暂停/或准备停止状态，请问原因是什么？
<img width="1393" alt="lQLPDhteG-DAN-zNBKDNCuKwKgfS3PEEKf4Catx4TwAFAA_2786_1184" src="https://user-images.githubusercontent.com/103353885/165258648-69be6688-d190-4522-b1b1-f96d4495eb16.png">

<img width="343" alt="lQLPDhteG_iXu5TNAg7NAq6w4JolsuBvwJwCatyff-5SAA_686_526" src="https://user-images.githubusercontent.com/103353885/165258530-f1bd8af2-56af-46c9-b631-3afaaa2a9e16.png">
<img width="684" alt="lQLPDhteHAIhJ2bNBB7NBViwcnAgv6bar70CatyvcgCGAA_1368_1054" src="https://user-images.githubusercontent.com/103353885/165258545-f8951923-6ad2-4f7f-90af-729869038887.png">

**Comment by SbloodyS** [comment, marked as answer]
which version you are using?

**Reply by 405594085** [reply]
2.0.3

**Reply by SbloodyS** [reply]
Could you check if there exists in 2.0.5? It seems like this #8323

**Comment by brave-lee** [comment]
This should be version 2.0.5, which has this bug
